tests/helm/const.go

```diff
@@ -5,14 +5,14 @@ const (
 	yamlDirectory            = "static"
 	chartDirectory           = "../../deploy/helm/sumologic"
 	chartName                = "sumologic"
-	releaseName              = "collection-test"
+	releaseName              = "col-test"
```

Comment:
Why is this change necessary?

Reply:
Else we get such failures: https://github.com/SumoLogic/sumologic-kubernetes-collection/actions/runs/6109112931/job/16579492782
and I found it easier to modify the constant instead of modifying the tests

Reply:
But why does this change cause these failures. `collection-test` is less than 22 characters, so it shouldn't make any difference, right?